pages/admin/[[...slug]].tsx

```diff
@@ -31,6 +31,7 @@ export const Editor = ({ client }: { client }) => {
   const prefix = "/admin";
   let slug = window.location.pathname.replace(prefix, "").slice(1);
 
+  const variables = util.variablesFromPath(slug, DEFAULT_VARIABLES);
```

Comment:
This allows me to pull `section` out for use in the `DocumentRenderer`.

Reply:
If you want `section`, you can pull it directly from the document at `getDocument -> sys -> section -> slug`:

```graphql
getDocument(section: $section, relativePath: $relativePath) {
  __typename
  ... on Pages_Document {
    sys {
      section {
        slug
      }
    }
    data {
      ...
    }
  }
}
```